src/types.db

```diff
@@ -175,6 +176,7 @@ ping_droprate           value:GAUGE:0:100
 ping_stddev             value:GAUGE:0:65535
 players                 value:GAUGE:0:1000000
 power                   value:GAUGE:0:U
+power_battery           value:GAUGE:U:U
```

Comment:
Uhm, we might have to change this then ;)

The existing code already (tries to) submit negative values:

```
[2016-09-07 09:22:43] uc_update: example.com/battery-0/power: ds[0] = -18.249000
```

Can you simply replace the minimum value of "power" with "U"? Thanks!